src/openzaak/components/autorisaties/api/viewsets.py

```diff
@@ -95,7 +95,14 @@ class ApplicatieViewSet(
     Na het verwijderen wordt een notificatie verstuurd.
     """
 
-    queryset = Applicatie.objects.prefetch_related("autorisaties").order_by("-pk")
+    queryset = (
+        Applicatie.objects.exclude(
+            heeft_alle_autorisaties=False, autorisaties__isnull=True
+        )
+        .exclude(heeft_alle_autorisaties=True, autorisaties__isnull=False)
```

Comment:
I'm not sure that invalid applicaties should be hidden for UPDATE or DELETE. It could cover cases, where the user creates invalid applicaties in the admin but then wants to change or delete them through API. 

Perhaps it's the reason why Postman tests are failing for this branch

Reply:
> I'm not sure that invalid applicaties should be hidden for UPDATE or DELETE. It could cover cases, where the user creates invalid applicaties in the admin but then wants to change or delete them through API.

I think you should either use the admin, or the API, not a mixture. The API rules prescribe that it's not allowed to have applications with superuser flag not set but also no authorizations. This expresses the expectation that you shouldn't see objects like that, ever. They are simply not "done" w/r to configuration.

> Perhaps it's the reason why Postman tests are failing for this branch
I'll have to look into that, thanks for the heads-up

Reply:
> Perhaps it's the reason why Postman tests are failing for this branch

This was indeed the case, but the root issue was a bit more nuanced. I believe commit 9e799673e186713face82724a18e59d0eff02c44 should fix it.